tools/update-deps

```diff
@@ -23,7 +23,7 @@ my $yamlfile     = "dependencies.yaml";
 my $file         = "$Bin/../$yamlfile";
 my $specfile     = "$Bin/../openQA.spec";
 my $cpanfile     = "$Bin/../cpanfile";
-my $circlecifile = "$Bin/../.circleci/dependencies.txt";
+my $circlecifile = "$Bin/../.circleci/ci_packages.txt";
```

Comment:
Just a comment: Checking this file in update-deps was an experiment, and it will be removed when we copy the current version from os-autoinst to openQA.